xml/libvirt_migrating_vms.xml

```diff
@@ -0,0 +1,623 @@
+    overloaded, the guests can be moved to another &vmhost;. &kvm; and &xen;
+    even support <quote>live</quote> migrations during which the &vmguest; is
+    constantly available.
+  </para>
+  <sect1 xml:id="libvirt-migration-types">
+    <title>Types of migration</title>
+
+    <para>
+      Depending on the required scenario, there are three ways you can migrate
+      virtual machines (VM).
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term>Live migration</term>
+        <listitem>
+          <para>
+            The source VM continues to run while its memory is transferred to
+            the target host. When the transfer is complete, the source VM is
+            suspended and the target VM is resumed.
+          </para>
+          <para>
+            Live migration is useful for VMs that need to be online without any
+            downtime.
+          </para>
+          <note>
+            <para>
+              VMs experiencing heavy I/O load or frequent memory page writes are
+              challenging to live migrate. In such cases, consider using
+              non-live or offline migration.
+            </para>
+          </note>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Non-live migration</term>
+        <listitem>
+          <para>
+            The source VM is suspended and its configuration and state
+            transferred to the target host. Then the destination VM is resumed.
+          </para>
+          <para>
+            Non-live migration is more reliable than live migration, although it
+            creates downtime for the VM. If downtime is tolerable, non-live
+            migration can be an option for VMs that are difficult to live
+            migrate.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Offline migration</term>
+        <listitem>
+          <para>
+            The VM definition is transferred to the target host. The source VM
+            is not stopped and the target VM is not resumed.
+          </para>
+          <para>
+            Offline migration can be used to migrate inactive VMs.
+          </para>
+          <important>
+            <para>
+              The <option>--persistent</option> option must be used together
+              with offline migration.
+            </para>
+          </important>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+  <sect1 xml:id="libvirt-admin-live-migration-requirements">
+    <title>Migration requirements</title>
+
+    <para>
+      To successfully migrate a &vmguest; to another &vmhost;, the following
+      requirements need to be met:
+    </para>
+
+    <important>
+      <para>
+        The post-copy live migration (not supported or recommended for
+        production) requires setting the
+        <varname>unprivileged_userfaultfd</varname> system value to
```

Comment:
A little bit off topic, but I don't think unprivileged_userfaultfd needs to be specified anymore. The kernel in SP6 and newer support /dev/userfaultfd. Fiddling with this sysctl knob is no longer required.